.github/contributing/INTEGRATIONS.md

```diff
@@ -142,9 +142,23 @@ This allows the linter and formatter to pick up example code blocks within docs
 
 As with all contributions, make sure you run `yarn lint` and `yarn format` so that everything conforms to our established style.
 
+### Separate integration packages
+
+While most integrations should generally reside in the `libs/langchain-community` workspace and be imported as `@langchain/community/module/name`, more in-depth integrations or suites of integrations may also reside in separate packages that depend on and extend `@langchain/core`. See [`@langchain/google-genai`](https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-google-genai) for an example.
+
+To make creating packages like this easier, we offer the [`create-langchain-integration`](https://github.com/langchain-ai/langchainjs/blob/main/libs/create-langchain-integration/) utility that will automatically scaffold a repo with support for both ESM + CJS entrypoints. You can run it like this:
```

Comment:
Add note saying it won't work if ran inside `langchain` repo

Reply:
Well we should fix quickly :)